src/badguy/badguy.cpp

```diff
@@ -442,6 +444,7 @@ BadGuy::collision_badguy(BadGuy& , const CollisionHit& )
 bool
 BadGuy::collision_squished(GameObject& object)
 {
+
```

Comment:
Did you forget the rumble_effect code here?

Reply:
No, it seems I accidentally added a newline and missed it when reviewing the diff. I'll amend the commit.